### DISCUSSION TITLE
When is the official version scheduled to be released?

### DISCUSSION THREAD

**Discussion by xungwoo** [rsocket/rsocket-js#236] [Q&A] [answered]
When is the official version, not alpha, scheduled to be released?

**Comment by viglucci** [comment, marked as answer]
Hey @xungwoo, thanks for the question. The short answer: we do not currently have a scheduled date.

Our intention/goal is to have larger consumers of `rsocket-js@0.27` exercise and provide feedback on the current `1.0.0-alpha` versions, however, we haven't received much feedback from current users just yet. We would like to address any such feedback before releasing `1.0.0`.

If the lack of a `1.0.0` version is a significant issue for you please do let us know if you would be willing to provide any feedback on the current alpha versions. Additionally, if you can disclose more about your use cases/where you hope to use `rsocket-js@1.0.0` that could help us with planning. Thanks

**Reply by xungwoo** [reply]
Thank you for answer.
I am considering applying rsocket-websocket to improve the product I am developing and is in the process of POC.
I will give you feedback while developing it ^^

**Reply by samburtonlaurie** [reply]
Hi @viglucci . Any word on a future official release of this, seems to be taking a long while? We're a bit hesitant about putting an alpha version in a production environment even if it has been published a long time now.

**Comment by sb000003** [comment]
Is anyone able to clarify when the 1.0 release will be "non alpha" ?
